### DISCUSSION TITLE
API for US_HI parser is returning nothing

### DISCUSSION THREAD

**Discussion by krisprice** [electricitymaps/electricitymaps-contrib#5328] [General]
Hi, I'm trying to fetch data for Oahu, so I was looking at the US_HI parser [here](https://github.com/electricitymaps/electricitymaps-contrib/blob/master/parsers/US_HI.py). But when I run it myself I get no results.

```
(base) C:\Code\sandpit>python ./islandpulse.py
fetch_production ->
Traceback (most recent call last):
  File "C:\Code\sandpit\islandpulse.py", line 109, in <module>
    print(fetch_production())
  File "C:\Code\sandpit\islandpulse.py", line 73, in fetch_production
    raw_data = res[0]
IndexError: list index out of range
```

For a shortcut this appears to be the API, but these return nothing.

- https://www.islandpulse.org/api/mix?date=2023-04-22
- https://www.islandpulse.org/api/mix?limit=1

Yet Electricity Maps is still showing updating data for Oahu. How is this working? (Or what am I doing wrong?)

**Comment by VIKTORVAV99** [comment]
It looks like the whole https://www.islandpulse.org/ page is more or less empty which seems to indicate a issue on their part.

As for why it is still showing up on Electricity Maps it's because we are estimating the zone when the live parser is unavailable.

**Reply by krisprice** [reply]
So it looks like the site has not been functional for a long time. I've just emailed the team at Sudokrew - since they seem to be the creators - to see if there is any plan to fix. Maybe you have a direct contact for them?

**Reply by VIKTORVAV99** [reply]
Sorry for not getting back to you earlier, as far as I know we don't have any direct connections with the team.

I'll monitor the site thought and see if the changes.

**Reply by krisprice** [reply]
No problem. I did reach out to Sudokrew. They told me they don't maintain the site. They built it for Blue Planet Foundation and to reach out to them about maintenance. Still haven't heard back from Blue Planet though. I'll ping them again then might see if I can reach someone via Hawaiian Electric.
